native/core/src/execution/operators/copy.rs

```diff
@@ -148,7 +148,7 @@ impl ExecutionPlan for CopyExec {
     }
 
     fn statistics(&self) -> DataFusionResult<Statistics> {
```

Comment:
should we also rename it to `partition_statistics` ?

Reply:
Updated